delivery_recipe/Discovery/discovery-findings.md

```diff
@@ -1,19 +1,30 @@
-# Discovery Findings
+# Discovery Outcomes
+
+The main outcome from discovery is the learning by doing the discovery. This outcome is more important than the output of documents.
+
+There will be some document outputs however. They'll depend on which discovery modules were undertaken, but the following will be the most common:
 
 ## Personas
 
 We start to build concise descriptions of the users of the service based on the research during discovery. These typically include clear goals, motivations, and contextual information that helps us think from their point of view when making decisons.
 
 ## User needs
 
-The high level goals, movitations, or problems to solve of the (potential) users of the service.
+The high level goals, motivations, or problems to solve of the \(potential\) users of the service. Good user needs are based on something a real user would say, and would avoid implying a solution.
 
-Good user needs are based on something a real user would say, and would avoid implying a solution.
+These are written in the format of [user stories](/delivery_recipe/help-cards/help-card-user-stories.md).
```

Comment:
I feel a bit unsure about linking user needs and stories so closely, they share the same format but their scope and usage are completely different, and I don't want people confusing the two.